wiki/pl.json

```diff
@@ -0,0 +1,14 @@
+{
+	"achievement-page-title": "Osiągnięcia",
+	"achievement-page-description": "Lista osiągnięć Manufactur'inc na Steam.",
+	"contracts-page-title": "Kontrakty",
+	"contract-page-title": "Kontrakt",
+	"equipments-page-title": "Equipments", //Review and context needed. What do you mean by equipment? Machines? 
```

Comment:
Yep, that's the word I use everywhere to include the machines and the storage. If you feel there's a better word in your language, feel free to use it (but we should try to have the same everywhere in the translations)

Reply:
Shoot... In this case we don't have the word for that. We separate words for machines and storage (a place or a thing that stores things) and it won't work the same as it does in English. Direct translation for "Equipment" to PL is "ekwipunek" which basically means stored items, not a thing that holds them per se but items in general that you have obtained. This one will be tricky to translate.

This is what ChatGPT had to say. He explained it better than I did:

In the Polish language, there isn’t a single word that directly refers to both machines and storage. However, you can use a more general term like **"infrastruktura"** (infrastructure), which can refer to resources, equipment, and storage in a broader context. Alternatively, you could consider using a combination of words, such as **"zaplecze techniczne"** (technical facilities) or **"zespół urządzeń i magazynów"** (set of equipment and warehouses), to convey the idea of both machines and storage.

Maybe we could use one of these words? Your choice

Reply:
I'm a bit scared too big words would make things harder.
Technical facilities seems okay ...

We can also talk about a more generic thing and go back to "buildings" or similar.

Maybe a translation of furnishings ?

Reply:
That could work. Let's add "wyposażenie" which is a word that describes what we need. Maybe in the future we could get some feedback from other Polish translators.

Reply:
I agree with "wyposażenie", and we'll see if other Polish-speaking peeps have other ideas :)